Hyper/Animation/Shaders/model_shader.vert

```diff
@@ -63,6 +67,6 @@ void main(void){
 
 	gl_Position = port(totalLocalPos * model) * view * projection;
 	FragPos = port(totalLocalPos * model);
-	Normal = totalNormal * TranslateMatrix(port(totalLocalPos * model));
+	Normal = totalNormal * normalRotation * TranslateMatrix(port(totalLocalPos * model));
```

Comment:
Are you sure we should have this? I think that normal rotation should already be accounter for in boneTransforms. I add a screenshot that shows that despite there being no lights behind the model its back is still bright. I don't think thats the case on tha main branch
![image](https://github.com/Non-Euclidean-World/Hyper/assets/95054664/43a0c86d-4806-47e7-8dc1-161bd9fba41c)

Reply:
That's funny because I added this for the sole purpose of fixing this very problem. This is what I get on main:
![image](https://github.com/Non-Euclidean-World/Hyper/assets/54417806/aae3f631-4da0-44f2-a68e-a5e773fe4845)
(obv no light source is behind the character),
and that's how it looks on my branch 
![image](https://github.com/Non-Euclidean-World/Hyper/assets/54417806/1c868fd2-d683-45a8-ab7c-a1d19957177b)

Reply:
ok. now i works i had an old version